apps/auction2/client.js

```diff
@@ -39,21 +42,17 @@ if (sd.AUCTION && sd.AUCTION.is_live_open == false) {
 }
 
 // Commercial filtering
-const params = new Params({sale_id: sd.AUCTION.id}, {
+const params = new Params({sale_id: sd.AUCTION.id, size: 20}, {
```

Comment:
Minor style note, but typically it's preferable to leave spaces between the curly braces, e.g. `({ sale_id: `. How do we feel about adding [standard](http://standardjs.com/) which also has an [automatic formatter](http://standardjs.com/#is-there-an-automatic-formatter) feature.